add-steps/haproxy.cfg

```diff
@@ -0,0 +1,26 @@
+defaults
+    mode    http
+    balance roundrobin
```

Comment:
I mean, it's always gotta be linkerd vs SOME load balancing algorithm.  Round robin is an easy strawman to defeat.